build.gradle.kts

```diff
@@ -111,8 +111,7 @@ allprojects {
 
         // The '-quiet' as second argument is actually a hack around https://github.com/gradle/gradle/issues/2354:
         // since the one-parameter `addStringOption` doesn't seem to work, we need to add an extra `-quiet`, which is added anyway by Gradle.
-        // options.addStringOption("Xwerror", "-quiet")
-        // options.addStringOption("Xdoclint:all", "-quiet")
+        options.jFlags("Xwerror", "Xdoclint:all")
```

Comment:
Oh! so probably now the above comment about the `-quitet` hack doesn't apply anymore